.github/workflows/check-archs-consistency.yml

```
@@ -0,0 +1,26 @@
+name: Test consistency between Paper & Fabric
+on: 
+  pull_request:
+    branches:
+      - main
+    paths: 
+      - src/specs/**
+      - android/paper/src/java/com/facebook/react/viewmanagers/**
```

```suggestion
      - android/paper/**
      - .github/workflows/check-archs-consistency.yml
    workflow_dispatch:
```

1. There's also a trubomodule interface in the `paper` directory
2. It would be nice not to have to modify the code files to test changes to the workflow file
3. `workflow_dispatch:` allows it to be run manually on any branch